static/gsAdmin/components/customers/customerOverview.spec.tsx

```diff
@@ -397,151 +392,36 @@ describe('CustomerOverview', function () {
     expect(screen.queryByText('Transactions:')).not.toBeInTheDocument();
   });
 
-  it('render product trials for am1 account', function () {
```

Comment:
instead of testing for every single tier, i think it's suffice to show that we render based on subscription state